package.json

```diff
@@ -25,7 +25,7 @@
     "express": "^4.9.7",
     "express-session": "^1.8.2",
     "lodash": "^3.5.0",
-    "mongoose": "^3.8.18",
```

Comment:
If this was `^3.8.18`, why not use `~3.8.18` (see [caret/tilde notation](http://stackoverflow.com/questions/22343224/difference-between-tilde-and-caret-in-package-json))? By specifying `3.8`, we accept pre-`3.8.18` to match.

(“we don't care because it won't happen” is an acceptable answer  ;)  )